pages/guides/agents/intermediate/_meta.json

```
@@ -58,5 +58,10 @@
     "title": "Building a RAG Agent",
     "tags": ["Intermediate", "Python", "Functions", "LangChain", "AI Engine"],
     "timestamp": true
+  },
+  "how-to-add-custom-rest-endpoints-to-your-agent": {
```

Title is too long, something like "rest endpoints" would be enough imo? 

```suggestion
  "rest-endpoints": {
```